client/asset/eth/multirpc.go

```diff
@@ -259,7 +259,7 @@ func (p *provider) subscribeHeaders(ctx context.Context, sub ethereum.Subscripti
 				return sub, nil
 			}
 			if time.Since(lastWarning) > 5*time.Minute {
-				log.Warnf("can't resubscribe to %q headers: %v", err)
+				log.Warnf("can't resubscribe to %q headers: %v", p.host, err)
```

Comment:
Separate commit please.

Reply:
This is not resolved.

Reply:
Do you mean separate PR?

Reply:
Which separate commit?  It's still part of the webserver commit

Reply:
This: https://github.com/decred/dcrdex/pull/2283/commits/4be67857b718fabc6a57a4fa4c6a12671a259a11

Reply:
Oh you touch this line in three separate commits now.
Please squash and rebase things, and start that commit with `client/asset/eth: `

Reply:
Okay.